pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -524,7 +535,16 @@ func TestImageConfigCreate(t *testing.T) {
 			f.expectGetMachineConfigAction(mcs2)
 			f.expectCreateMachineConfigAction(mcs2)
 
-			f.run("cluster")
+			c := f.newController()
```

Comment:
sorry, could you clarify what this test change is trying to do?

Reply:
Yeah, with `f.run("cluster")` it was calling the sync function for both containerRuntimeConfig and Image. This was causing the test to fail as it was expecting the actions for creating the capabilities. But since this function is only testing changes to an Image CR, I changed it to just run the Image sync function, similar to how the other Image tests are doing. I think this was accidentally left over from when the Image tests were modified.